lib/colors/package.d

```diff
@@ -2,6 +2,7 @@ module raylib_misc.colors;
 public
 {
     import raylib_misc.colors.translucent;
+    import raylib_misc.colors.adwaita;
```

Comment:
I'm fine with the change but these imports were in alphabetical order before. Is there a reason for messing with the order now?

It would make it easier to look for imports as more colour palettes are added over time if the imports were in alphabetical order.

```suggestion
     import raylib_misc.colors.adwaita;
     import raylib_misc.colors.translucent;
```

Reply:
see coding style .9:

>  the longer lines should be on top and the smaller lines should be at the bottom.